lib/builder/webpack/client.js

```diff
@@ -164,6 +144,27 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
       }
     }
 
+    // Make uglifyjs faster
```

Comment:
Thank you for you contribution！
Could you explain why move down the plugin ？

Reply:
Because source-map usually ( or at least can be ) added in build extend. E.g. to only build it for client. It is even showed in nuxt documentation https://nuxtjs.org/api/configuration-build/#extend
So i moved it down to be after nuxt evaluate extend function.

Reply:
Make sense, `!config.optimization.minimizer` check will also prevent overriding user changes in `minimizer`.

And even user deleted `minimizer` in extend, webpack will still give it a default one, we actually do the same thing but with more configs.